src/test/java/uk/gov/hmcts/reform/em/stitching/service/impl/DmStoreDownloaderImplTest.java

```diff
@@ -53,6 +50,37 @@ private static Response intercept(Interceptor.Chain chain) throws IOException {
             .build();
     }
 
+    private static Response interceptException(Interceptor.Chain chain) {
+
+        return new Response.Builder()
+                .request(chain.request())
+                .message("")
+                .code(400)
+                .protocol(Protocol.HTTP_2)
+                .build();
+    }
+
+    @Test
+    public void downloadFileException() {
+        http = new OkHttpClient
+                .Builder()
+                .addInterceptor(DmStoreDownloaderImplTest::interceptException)
+                .build();
+
+        dmStoreDownloader = new DmStoreDownloaderImpl(http, () -> "auth", dmStoreUriFormatter);
+
+        BundleDocument mockBundleDocument1 = new BundleDocument();
+        BundleDocument mockBundleDocument2 = new BundleDocument();
+        mockBundleDocument1.setDocumentURI("/AAAA");
+        mockBundleDocument2.setDocumentURI("/BBBB");
+        try {
+            dmStoreDownloader.downloadFiles(Stream.of(mockBundleDocument1, mockBundleDocument2));
+        } catch (DocumentTaskProcessingException ex) {
+            Assert.assertTrue(ex instanceof DocumentTaskProcessingException);
```

Comment:
this assertion should always be made right? Currently, if the try stuff works, it'll pass the test, although its not testing wha tit should test.

Reply:
You could run it without a try catch, and l63 would say `@Test(expected = DocumentTaskProcessingException)` or something like that